doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -87,7 +117,12 @@ the source code, build it, and install it on your device.
 
          sudo umount /boot/efi
          sudo lsblk
+         
+         # For SATA
          sudo mount /dev/sda1 /mnt
```

Comment:
Isn't the ESP automatically mounted under `/boot` in Ubuntu? (it is the case in Clear Linux for example, and under `/boot/efi` in Fedora). This would (slightly) simplify the instructions.

Reply:
No, this is little different with Clear Linux, even in Clear Linux, this is not case in past,